server/plugin.go

```diff
@@ -57,6 +59,22 @@ func (p *Plugin) OnActivate() error {
 	}
 	p.i18nBundle = i18nBundle
 
+	jitsiBot := &model.Bot{
+		Username:    "jitsi",
+		DisplayName: "Jitsi",
+		Description: "A bot account created by the jitsi plugin",
+	}
+	options := []plugin.EnsureBotOption{
+		plugin.ProfileImagePath("assets/icon_square.png"),
```

Comment:
Maybe use `icon.png` as filename to indicate that it's the same as https://github.com/mattermost/mattermost-plugin-jitsi/blob/master/assets/icon.svg?

Reply:
Done :white_check_mark: